### DISCUSSION TITLE
Using an external keyboard(mech) with a laptop

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1575] [Questions] [answered]
First of all, you must be wondering why I am posting this on GitHub, here's my answer: I sadly don't have a phone making it not possible for me to join the Monkeytype discord server to ask my question because I can't do phone verification unless it's possible using Android Studio's emulator when I'm not making anything, so I thought why not ask my question here and hopefully get advice from an awesome typer like you. This is a question I've pondered about for several weeks. I currently use a chromebook for most of my work(typing on Monkeytype and coding :) ) but I wouldn't say the typing experience is the best with the membrane laptop keys. Recently I was given an Amazon gift card and am unsure on what to use it for. I settled for a mech. The problem is, I don't have a desktop to use it with (only my school issued chromebook and an [overkill gaming laptop](https://www.amazon.com/MSI-GE66210-Raider-i7-10870H-RTX3080/dp/B08R9GGMBS/ref=sr_1_3?dchild=1&keywords=msi+ge66+raider+rtx+3080&qid=1624570428&sr=8-3) I can sometimes get access to) nor what keyboard I should buy. Do you think it is a good idea to use a mech with a laptop (yes/no and pros and cons) and do you have any advice for a first time mech buyer? Thanks in advance.

**Comment by Miodec** [comment, marked as answer]
Send me a dm on discord ill sort out the phone verification you can ask all of us there
